package/MDAnalysis/lib/transformations.py

```diff
@@ -970,8 +978,12 @@ def superimposition_matrix(v0, v1, scaling=False, usesvd=True):
     True
 
     """
-    v0 = np.array(v0, dtype=np.float64, copy=False)[:3]
-    v1 = np.array(v1, dtype=np.float64, copy=False)[:3]
+    if np.__version__[0] == "2":
```

Comment:
Do we want to wrap this in a helper routine to avoid repetition eg:

```python
 v0 = np.array(v0, dtype=np.float64, copy=no_copy_shim())[:3]


def no_copy_shim():
      if np.__version__[0] == "2":
        copy=None
     else:
        copy=False
...
```

Ill leave it up to you @tylerjereddy?